packages/components/tooltip/src/tooltip.scss

```diff
@@ -1,28 +1,133 @@
 :host {
-  --_background: #fff;
+  --_background: var(--sl-color-tooltip-background);
   --_border: 0;
-  --_border-radius: 3px;
-  --_box-shadow: 0 6px 10px -6px rgba(0, 0, 0, 0.7);
-  --_color: #333;
-  --_padding: 0.5rem 0.75rem;
+  --_border-radius: var(--sl-border-radius-default);
+  --_box-shadow: var(--sl-box-shadow-tooltip);
+  --_color: var(--sl-color-tooltip-foreground);
+  --_font: var(--sl-text-tooltip-text-tip);
+  --_max-width: var(--sl-tooltip-max-width, auto);
+  --_padding-block: var(--sl-space-tooltip-block);
+  --_padding-inline: var(--sl-space-tooltip-inline);
+
+  /* to position */
+  --_arrow-height: 6px;
+  --_arrow-width: 16px;
+  /* the distance from the edge when the arrow is positionend at start or end */
+  --_arrow-start-end-offset: var(--sl-space-tooltip-arrow-offset);
 }
 
 :host {
-  // This has to be in pixels until we can use @property
-  --sl-popover-offset: 8px;
+  /** This has to be in pixels until we can use @property.
```

Comment:
Perhaps check if we can use `@property` with a fallback for FF/older browsers? https://caniuse.com/mdn-css_at-rules_property

Reply:
Should probably be done in a different PR.

Reply:
Created a story for this: #656